src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -65,6 +69,8 @@ public CommentAttributes(Comment comment) {
         this.recipients = comment.getRecipients();
         this.createdAt = comment.getCreatedAt();
         this.commentText = comment.getCommentText();
+        this.lastEditorEmail = comment.getLastEditorEmail();
+        this.lastEditedAt = comment.getLastEditedAt();
```

Comment:
Does it impact anything if without the boolean param?
eg every time when update a comment, it will always update last edit info